DirectProgramming/C++SYCL_FPGA/Tutorials/Tools/use_library/README.md

```diff
@@ -114,141 +151,146 @@ Note that the library files (\*.a) must be included after all of the cpp files i
+   >
+   > **Note**: The sample is defaultly targeting device Agilex family in `lib_rtl_dsp_spec.xml`. You will need to manually change the parameter in `lib_rtl_dsp_spec.xml` if you wish to target other devices.
+
+3. Compile the design. (The provided targets match the recommended development flow.)
+
+   1. Compile and run for emulation (fast compile time, targets emulates an FPGA device).
+      ```
       make fpga_emu
       ```
-
-   * Generate the optimization report:
-
-     ```bash
-     make report
-     ```
-
-   * Compile for FPGA Simulator
-
-     ```bash
-     make fpga_sim
-     ```
-
-   * Compile for FPGA hardware (longer compile time, targets FPGA device):
-
-     ```bash
-     make fpga
-     ```
+   2. Generate the HTML optimization reports. (See [Read the Reports](#read-the-reports) below for information on finding and understanding the reports.)
+      ```
+      make report
+      ```
+   3. Compile for simulation (fast compile time, targets simulated FPGA device).
+      ```
+      make fpga_sim
+      ```
+   4. Compile and run on FPGA hardware (longer compile time, targets an FPGA device).
+      ```
+      make fpga
+      ```
 
 ### On a Windows* System
 
-1. Generate the `Makefile` by running `cmake`.
-
-    ```
-    mkdir build
-    cd build
-    ```
-
-    To compile for the default target (the Agilex® 7 device family), run `cmake` using the command:
-    ```
-    cmake -G "NMake Makefiles" ..
-    ```
-    > **Note**: You can change the default target by using the command:
-    >  ```
-    >  cmake -G "NMake Makefiles" .. -DFPGA_DEVICE=<FPGA device family or FPGA part number>
-    >  ```
-    >
-    > Alternatively, you can target an explicit FPGA board variant and BSP by using the following command:
-    >  ```
-    >  cmake -G "NMake Makefiles" .. -DFPGA_DEVICE=<board-support-package>:<board-variant>
-    >  ```
-    >
-    > You will only be able to run an executable on the FPGA if you specified a BSP.
-
-2. Compile the design through the generated `Makefile`. The following build targets are provided, matching the recommended development flow:
-
-   - Compile for emulation (fast compile time, targets emulated FPGA device):
-
-     ```
-     nmake fpga_emu
-     ```
-
-   - Generate the optimization report:
+1. Change to the sample directory.
+2. Build the program for the Intel® Agilex® 7 device family, which is the default.
+   ```
+   mkdir build
+   cd build
+   cmake -G "NMake Makefiles" ..
+   ```
+   > **Note**: You can change the default target by using the command:
+   >  ```
+   >  cmake -G "NMake Makefiles" .. -DFPGA_DEVICE=<FPGA device family or FPGA part number>
+   >  ```
+   >
+   > Alternatively, you can target an explicit FPGA board variant and BSP by using the following command:
+   >  ```
+   >  cmake -G "NMake Makefiles" .. -DFPGA_DEVICE=<board-support-package>:<board-variant>
+   >  ```
+   >
+   > You will only be able to run an executable on the FPGA if you specified a BSP.
```

Comment:
```suggestion
```
removed multiarchitecture binary kernel support for this design for better area

Reply:
?